Networking/Networking/Model/WCPayAccountStatusEnum.swift

```diff
@@ -0,0 +1,100 @@
+    case rejectedTermsOfService
+
+    /// rejectedListed: This state occurs when our payment processor rejects the merchant account due to sanctions/being
+    /// on a watch list. The merchant will NOT be able to collect card present payments.
+    case rejectedListed
+
+    /// rejectedListed: This state occurs when our payment processor rejects the merchant account due to any other reason.
+    /// The merchant will NOT be able to collect card present payments.
+    case rejectedOther
+
+    /// noAccount: This state occurs when the merchant hasn't started on-boarding yet. The merchant will NOT be able to
+    /// collect card present payments.
+    case noAccount
+
+    /// unknown: This state occurs when the self-hosted site responded in a way we don't recognize.
+    case unknown
+}
+
+/// RawRepresentable Conformance
+///
+extension WCPayAccountStatusEnum: RawRepresentable {
+
+    /// Designated Initializer. Takes the string in the response and returns the enum
+    /// assocated with that value. Empty strings return .noAccount
+    /// If not recognized, returns .unknown
+    ///
+    public init(rawValue: String) {
+        switch rawValue {
+        case Keys.complete:
+            self = .complete
+        case Keys.restricted:
+            self = .restricted
+        case Keys.rejectedFraud:
+            self = .rejectedFraud
+        case Keys.rejectedTermsOfService:
+            self = .rejectedTermsOfService
+        case Keys.rejectedListed:
+            self = .rejectedListed
+        case Keys.rejectedOther:
+            self = .rejectedOther
+        case Keys.noAccount:
+            self = .noAccount
+        default:
+            self = .unknown
+        }
+    }
+
+    /// Returns the current Enum Case's raw value
+    ///
+    public var rawValue: String {
+        switch self {
+        case .complete:               return Keys.complete
+        case .restricted:             return Keys.restricted
+        case .rejectedFraud:          return Keys.rejectedFraud
+        case .rejectedTermsOfService: return Keys.rejectedTermsOfService
+        case .rejectedListed:         return Keys.rejectedListed
+        case .rejectedOther:          return Keys.rejectedOther
+        case .noAccount:              return Keys.noAccount
+        case .unknown:                return Keys.unknown
+        }
+    }
+}
+
+/// Enum containing all possible account status keys
+///
+private enum Keys {
+    static let complete               = "complete"
+    static let restricted             = "restricted"
+    static let rejectedFraud          = "rejected.fraud"
+    static let rejectedTermsOfService = "rejected.terms_of_service"
+    static let rejectedListed         = "rejected.listed"
+    static let rejectedOther          = "rejected.other"
+    static let noAccount              = "NOACCOUNT" /// The status field will be not present in the API response when no account has been set up.
```

Comment:
Maybe you would like to consider moving the comment above so that it's visible in Xcode's helper. 🙂 

```suggestion
    /// The status field will be not present in the API response when no account has been set up.
    static let noAccount              = "NOACCOUNT" 
```

<img src="https://user-images.githubusercontent.com/198826/113335908-d6dcf080-92e2-11eb-8ab9-9a3f57333906.png" width="500">

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3881/commits/a544444a49d41e5f8ba83d906ac36435bd614a93